app/views/unattended/provisioning_templates/snippet/create_users.erb

```diff
@@ -3,6 +3,8 @@ name: create_users
 model: ProvisioningTemplate
 kind: snippet
 snippet: true
+description: |
+  This snippet can be used  
```

Comment:
Absolutely :-D

However, it looks like it is not used anywhere in templates.

Reply:
Maybe this is used by ReX @adamruzicka ? Can you provide description in that case?

Reply:
I've finished the description, it's quite clear :-)